js/index.js

```diff
@@ -1,42 +1,109 @@
 // ITERATION 1
+  // ITERATION 2
+  //... your code goes here
+  function updateSubtotal(product) {
+    let price = product.querySelector(".price span");
+    let quantity = product.querySelector(".quantity");
+    let priceVal = price.innerHTML;
+    let quantityVal = product.querySelector("input").value;
+    let subtotalVal = priceVal * quantityVal;
+    let subtotal = product.querySelector(".subtotal span");
+    subtotal.innerHTML = subtotalVal;
+    //return subtotalVal;
```

Comment:
Please remove unnecessary comments/code to keep your code as minimal and readable as possible.